buckets/archive/tracker.go

```diff
@@ -22,8 +23,7 @@ const (
 )
 
 var (
-	CheckInterval         = time.Second * 15
-	JobStatusPollInterval = time.Minute * 30
```

Comment:
This is now configured and passed in here

Reply:
Nice!